test/verify/check-login

```diff
@@ -169,6 +169,44 @@ account    required     pam_succeed_if.so user ingroup %s""" % m.get_admin_group
 
         self.allow_restart_journal_messages()
 
+
+    @skipImage("logs in via ssh, not cockpit-session", "fedora-coreos")
+    def testLogging(self):
+        m = self.machine
+        b = self.browser
+
+        # Clean out the relevant logfiles
+        m.execute("truncate -s0 /var/log/{[bw]tmp,lastlog} /var/run/utmp")
+
+        m.start_cockpit()
+        b.login_and_go('/system')
+        time.sleep(2)
+        assert not b.is_present('#last-login')
+        assert not b.is_present('#last-failed')
+        b.logout()
+
+        b.login_and_go('/system')
+        b.wait_present('#login-messages')
+        assert b.is_visible('#last-login')
```

Comment:
`b.wait_visible('#last-login')`

Reply:
If the login message are present then this should already be there, from the start.  Why wait for it?

Reply:
Because that's how React works.. Pretty well nothing is synchronous.